<ui version="4.0" >
 <class>GeneralSettingsWidgetUI</class>
 <widget class="QWidget" name="GeneralSettingsWidgetUI" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>460</width>
    <height>550</height>
   </rect>
  </property>
  <property name="minimumSize" >
   <size>
    <width>460</width>
    <height>550</height>
   </size>
  </property>
  <property name="windowTitle" >
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2" >
   <item row="0" column="0" >
    <widget class="QGroupBox" name="groupBox" >
     <property name="minimumSize" >
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize" >
      <size>
       <width>16777215</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="title" >
      <string>General Settings</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_3" >
      <item row="0" column="0" >
       <layout class="QVBoxLayout" name="verticalLayout" >
        <item>
         <widget class="QCheckBox" name="kcfg_overwriteExistingFiles" >
          <property name="text" >
           <string>Overwrite existing files</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="kcfg_deletePartialFiles" >
          <property name="text" >
           <string>Delete partial files after cancel</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="kcfg_ejectCDTray" >
          <property name="text" >
           <string>Eject CD tray after extracting finished</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="kcfg_cddbLookupAuto" >
          <property name="text" >
           <string>Perform CDDB lookup automatically</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0" >
    <widget class="QGroupBox" name="groupBox_1" >
     <property name="sizePolicy" >
      <sizepolicy vsizetype="Fixed" hsizetype="Preferred" >
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize" >
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize" >
      <size>
       <width>16777215</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="baseSize" >
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="title" >
      <string>Localizations</string>
     </property>
     <layout class="QGridLayout" name="gridLayout" >
      <item row="0" column="0" >
       <layout class="QHBoxLayout" >
        <item>
         <widget class="QLabel" name="label_amazonLocale" >
          <property name="minimumSize" >
           <size>
            <width>80</width>
            <height>0</height>
           </size>
          </property>
          <property name="text" >
           <string>Amazon</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="KComboBox" name="kcfg_amazonLocale" >
          <property name="minimumSize" >
           <size>
            <width>150</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize" >
           <size>
            <width>150</width>
            <height>16777215</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <spacer>
          <property name="orientation" >
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0" >
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item row="1" column="0" >
       <layout class="QHBoxLayout" name="horizontalLayout" >
        <item>
         <widget class="QLabel" name="label_wikipediaLocale" >
          <property name="minimumSize" >
           <size>
            <width>80</width>
            <height>0</height>
           </size>
          </property>
          <property name="text" >
           <string>Wikipedia</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="KComboBox" name="kcfg_wikipediaLocale" >
          <property name="minimumSize" >
           <size>
            <width>150</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize" >
           <size>
            <width>150</width>
            <height>16777215</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer" >
          <property name="orientation" >
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0" >
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0" >
    <widget class="QGroupBox" name="groupBox_2" >
     <property name="minimumSize" >
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize" >
      <size>
       <width>16777215</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="title" >
      <string>Device</string>
     </property>
     <layout class="QGridLayout" >
      <item row="0" column="0" >
       <layout class="QHBoxLayout" >
        <item>
         <widget class="KLineEdit" name="kcfg_cdDevice" >
          <property name="showClearButton" stdset="0" >
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <spacer>
          <property name="orientation" >
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0" >
           <size>
            <width>91</width>
            <height>23</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item row="1" column="0" >
       <widget class="QCheckBox" name="kcfg_paranoiaMode" >
        <property name="text" >
         <string>Full paranoia mode (best quality)</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0" >
       <widget class="QCheckBox" name="kcfg_neverSkip" >
        <property name="text" >
         <string>Never skip on read error</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="3" column="0" >
    <widget class="QGroupBox" name="groupBox_3" >
     <property name="maximumSize" >
      <size>
       <width>16777215</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="title" >
      <string>Paths</string>
     </property>
     <layout class="QGridLayout" >
      <item row="0" column="0" >
       <layout class="QVBoxLayout" >
        <item>
         <widget class="QLabel" name="label_tempPath" >
          <property name="text" >
           <string>Temporary path</string>
          </property>
          <property name="buddy" >
           <cstring>urlreq_tempPath</cstring>
          </property>
         </widget>
        </item>
        <item>
         <widget class="KUrlRequester" name="urlreq_tempPath" />
        </item>
       </layout>
      </item>
      <item row="1" column="0" >
       <layout class="QVBoxLayout" >
        <item>
         <widget class="QLabel" name="label_basePath" >
          <property name="text" >
           <string>Base path to store music files</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="KUrlRequester" name="urlreq_basePath" />
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KComboBox</class>
   <extends>QComboBox</extends>
   <header>kcombobox.h</header>
  </customwidget>
  <customwidget>
   <class>KLineEdit</class>
   <extends>QLineEdit</extends>
   <header>klineedit.h</header>
  </customwidget>
  <customwidget>
   <class>KUrlRequester</class>
   <extends>QFrame</extends>
   <header>kurlrequester.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
